hardhat/env-artifacts.js

```diff
@@ -0,0 +1,20 @@
+const { HardhatError } = require('hardhat/internal/core/errors');
+
+extendEnvironment(env => {
```

Comment:
IMO, there should be a comment explaining what this actually does and why its needed.